rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -37,6 +38,12 @@
 from random import randrange
 
 
+def verify_clusters_per_vpc(clusters):
```

Comment:
This could be repleaced with a choices=range(1, 10) in the parser.add_argument

Reply:
I like the idea, changed!